bin/tmp/initial-device-tests-html.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
If I understand it correctly, the files in the `bin/tmp` folder are not exactly temporary or anything. Instead, they are the different source files to use to overwrite the `src/app/initial-html.js` module. This is nit but, can we rename the `tmp` folder to something else, maybe `example-content` or similar?

Reply:
That's correct, we can rename it to a more appropriate name

Reply:
Seems addressed by 132202a7f809f14eef26ab6e0ba2f33da72a0e19.